cinema/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Currently, no models are registered with the Django admin site. If you plan to use the admin interface, consider registering your models here using `admin.site.register(YourModel)`. This will allow you to manage your models through the Django admin panel.